docs/source/developer_guide/build.md

```diff
@@ -18,30 +18,49 @@ git clone https://github.com/rapidsai/cuspatial.git $CUSPATIAL_HOME
 2. clone the cuSpatial repo
 
 ```shell
-conda env update --file conda/environments/all_cuda-118_arch-x86_64.yaml
+conda env create -n cuspatial --file conda/environments/all_cuda-118_arch-x86_64.yaml
 ```
 
-## Build and install cuSpatial
+## Build cuSpatial
 
-1. Compile and install
-   ```shell
-   cd $CUSPATIAL_HOME && \
-   chmod +x ./build.sh && \
-   ./build.sh
-   ```
+### From the cuSpatial Dev Container:
 
-2. Run C++/Python test code
+Execute `build-cuspatial-cpp to build `libcuspatial`. The following options may be added.
+ - `-DBUILD_TESTS=ON`: build `libcuspatial` unit tests.
+ - `-DBUILD_BENCHMARKS=ON`: build `libcuspatial` benchmarks.
+ - `-DCMAKE_BUILD_TYPE=Debug`: Create a Debug build of `libcuspatial` (default is Release).
+In addition, `build-cuspatial-python` to build cuspatial cython components.
 
-   Some tests using inline data can be run directly, e.g.:
+### From Bare Metal:
 
-   ```shell
-   $CUSPATIAL_HOME/cpp/build/gtests/LEGACY_HAUSDORFF_TEST
-   $CUSPATIAL_HOME/cpp/build/gtests/POINT_IN_POLYGON_TEST
-   python python/cuspatial/cuspatial/tests/legacy/test_hausdorff_distance.py
-   python python/cuspatial/cuspatial/tests/test_pip.py
-   ```
+Compile libcuspatial (C++), cuspatial (cython) and C++ tests:
+```shell
+cd $CUSPATIAL_HOME && \
+chmod +x ./build.sh && \
+./build.sh libcuspatial cuspatial tests
+```
+Additionally, the following options are also commonly used:
+- `benchmarks`: build libcuspatial benchmarks
+- `clean`: remove all existing build artifacts and configuration
+Execute `./build.sh -h` for full list of available options.
+
+## Validate Installation with C++ and Python Tests
+
+- C++ tests are located within the `$CUSPATIAL_HOME/cpp/build/gtests` directory.
```

Comment:
Where does `build.sh` put the build? For dev container, need latest on path...

```suggestion
- C++ tests are located within the `$CUSPATIAL_HOME/cpp/build/gtests/latest` directory.
```

Reply:
`latest` should go before `gtests`.